AgentDiscoveries-Backend/src/main/java/org/softwire/training/api/core/CustomisedGsonBuilder.java

```diff
@@ -17,15 +15,35 @@
 public class CustomisedGsonBuilder {
 
     public static Gson getGson() {
+
+        ExclusionStrategy strategy = new ExclusionStrategy() {
```

Comment:
Nice use of this exclusion strategy - particularly noting that it wants to be serialization only so that deserializing still works!